test/packages/logs_synthetic_mode/data_stream/synthetic/fields/some-fields.yml

```diff
@@ -1,3 +1,3 @@
 - name: decision_id
-  type: text
+  type: long
```

Comment:
`store` is set to true by default for `text` fields since 8.18. Testing this setting with a `long` field now.